.github/workflows/deploy-gcp-tests.yml

```diff
@@ -1362,15 +1434,3 @@ jobs:
           else
             gcloud compute instances delete "${INSTANCE}" --zone "${{ env.ZONE }}" --delete-disks all --quiet
           fi
-
-      # Deletes SSH keys generated during this workflow run, as GCP has a limit of SSH keys
-      # that can exist at the same time in the OS Login metadata. Not deleting this keys
-      # could cause the following error:
-      # `Login profile size exceeds 32 KiB. Delete profile values to make additional space`
-      - name: Delete temporal SSH keys
```

Comment:
Will outdared PRs continue to create these keys until they are all rebased on this PR? 
How will we do a final cleanup of these unused keys?

Do we need to keep on running this step until the last PR has been updated?

Reply:
This were specific keys created by the OS-Login function which was added project-wide on GCP. This is no longer happening, and all keys have been deleted.

We would anyways need to rebase all PRs to stop SSH errors from happening